src/Component/Field.js

```diff
@@ -0,0 +1,275 @@
+        switch(e.target.type){
+          case 'button': 
+             handlerButtonTypeButton();
+             break;
+           case 'submit': 
+             handleButtonTypesubmit();
+             break;
+           case "reset":
+             handleButtonTypeReset();
+             break;
+        }
+   }
+
+    function handleOnchange(e,field) {
+        setText(e.target.value)
+    } 
+
+    function handleBulr(e,field){ 
+      console.log(e.target)
+      //apply validation here 
+        if(e.target.value==="") return
+        if(fieldValue.length===0) return setFieldValue([...fieldValue,{name:field.attributes.name,value:e.target.value}])
+        if(fieldValue.length>0){
+            if(fieldValue.find(field => field.name===e.target.value)===undefined){
+              setFieldValue([...fieldValue,{name:e.target.name,value:e.target.value}])
+            }
+            const tempArr=fieldValue.filter(field => field.name!==e.target.name)
+            tempArr.push({name: e.target.name,value: e.target.value})
+            setFieldValue(tempArr)
+        }
+    }
+
+    function createLabel(field) {
+        return element = (<label>{field.attributes.value}</label>)
+        }
+     
+        function createTextArea(field){
+           return element = (<textarea 
+                               name={field.attributes.name}
+                               rows={field.attributes.rows}
+                               cols={field.attributes.cols}
+                               ></textarea>)
+        }
+     
+       function createButton(field){
+       return element = (<button
+                           type={field.fieldConfig.type}
+                           formAction={field.attributes.formAction}
+                           formEncType={field.attributes.formEncType}
+                           formMethod={field.attributes.formMethod}
+                           formTarget={field.attributes.target}
+                           onClick={(e) => fieldClickhandler(e)}
+                           >
+                           {field.attributes.value}
+                         </button>)
+       }
+     
+
+       function onChangeSelectHanlder(e){
+            console.log(e.target.value);
+       }
+
+       function onBlueSelectHandler(e){
+         console.log("On Blue",e.target.value)
+       }
+
+       function createSelectField(field){
+           return element = (
+                       <select 
+                         name={field.attributes.name} 
+                         onChange={(e) => handleOnchange(e)} 
+                         onBlur={(e) => handleBulr(e)}>
+                           {field.options.map(option => {
+                               return(
+                                <option 
+                                    key={option.key} 
+                                    value={option.value}>
+                                  {option.displayName}
+                            </option>
```

Comment:
Keep options structure like below as it's highly accepted:

`[{label: 'Option 1', value: 1}]`

Reply:
ohk